Program/stud_fees_data.py

```diff
@@ -0,0 +1,58 @@
+#Program [ 1 ]
+#Create table stud(id, name, dept, paidfee) with 8 records.
+#Create stud.csv file of the stud data (Department will be BBA and BCA).
+#Read csv file using reader object and write data with remaining fee.
+#( Total fee of bba is 13400 and bca is 15400 Rs)
+
+import csv
+

```

Comment:
Can't we access csv file by importing reader only object?

Reply:
> Can't we access csv file by importing reader only object?

Yes we can. There is also another way to import csv file for reading by using " from csv import reader "